### ISSUE TITLE
product_where with Typoscript

### ISSUE THREAD

**Issue by webian** [franzholz/tt_products#37]
We can set `product_where` in BE (flexform) but, if I'm not wrong, we can't set it with Typoscript and it could be very useful.

By instance, using Typoscript, while showing the SINGLE view of a product (uid=123), could be useful to display also the other products (LIST) present in this category but not display the current product in this list (WHERE uid != 123)

**Comment by franzholz** [comment]
Yes, you have the filter setup which gives you many more possibilities with the support for regexp.

**Comment by webian** [comment]
In this use case I need a product list just after the product detail that displays other products (except current one).

Unfortunately, with current tt_products filters, it's not possible to add a condition like `uid != {GP:tt_products|product}` to filter out current product because the TS parameter is not passed to stdWrap.

**Comment by franzholz** [comment]
For this use case there exists already the related and the accessory products lists.

 And you can already use this setup:

    plugin.tt_products.conf.tt_products.LISTRELATED.filter.where.field {
        uid NOT IN (123)
    }

Maybe this setup does not work yet and needs improvement.
However it still would not help you, because the value of uid varies for each product.
Can the placeholder be used here?  `{GP:tt_products|product}`
